components/org.wso2.carbon.identity.branding.preference.management.core/src/main/java/org/wso2/carbon/identity/branding/preference/management/core/constant/BrandingPreferenceMgtConstants.java

```diff
@@ -54,7 +54,7 @@ public enum ErrorMessages {
         ERROR_CODE_INVALID_BRANDING_PREFERENCE("BRANDINGM_00001",
                 "Invalid Branding Preference configurations for tenant: %s."),
         ERROR_CODE_BRANDING_PREFERENCE_NOT_EXISTS("BRANDINGM_00002",
-                "Branding preferences are not configured for tenant: %s."),
+                "Branding preferences are not configured for name: %s, type: %s, tenant: %s."),
```

Comment:
Try to go for a new constant

Reply:
Addressed with https://github.com/wso2-extensions/identity-branding-preference-management/pull/40/commits/05fa6da03cb482ecbd514d7030744011bad7a98e